src/StorageSync/StorageSync/help/Invoke-AzStorageSyncChangeDetection.md

```diff
@@ -0,0 +1,308 @@
+```
+
+In this example, change detection is run in the "Data" and "Reporting\Templates" directories of a syncing Azure file share. All paths are relative to the root of the Azure file share namespace.
+
+### Example 2
+```powershell
+PS C:\> Invoke-AzStorageSyncChangeDetection -ResourceGroupName "myResourceGroup" -StorageSyncServiceName "myStorageSyncServiceName" -SyncGroupName "mySyncGroupName" -CloudEndpointName "myCloudEndpointName" -Path "Data\results.xslx","Reporting\Templates\generated.pptx"
+```
+
+In this example, change detection is run for a set of files that are known to the command caller to have changed. The goal is to have Azure file sync detect and sync these changes as well.
+
+### Example 3
+```powershell
+PS C:\> Invoke-AzStorageSyncChangeDetection -ResourceGroupName "myResourceGroup" -StorageSyncServiceName "myStorageSyncServiceName" -SyncGroupName "mySyncGroupName" -CloudEndpointName "myCloudEndpointName" -DirectoryPath "Examples" -Recursive
+```
+
+In this example, change detection is run for the "Examples" directory and will recursively detect changes in subdirectories. 
+Keep in mind that this command can detect up to 10,000 changes before it is automatically aborted. If you suspect more than 10,000 changes to have occurred, run the command on sub-parts of the namespace.
+
+## PARAMETERS
+
+### -AsJob
+Run cmdlet in the background
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -DefaultProfile
+The credentials, account, tenant, and subscription used for communication with Azure.
+
+```yaml
+Type: IAzureContextContainer
+Parameter Sets: (All)
+Aliases: AzContext, AzureRmContext, AzureCredential
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -DirectoryPath
+Directory where change detection will be performed.
+
+```yaml
+Type: String
+Parameter Sets: StringAndDirectoryParameterSet, ResourceIdAndDirectoryParameterSet, ObjectAndDirectoryParameterSet
+Aliases:
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -InputObject
+CloudEndpoint Object, normally passed through the parameter.
+
+```yaml
+Type: PSCloudEndpoint
+Parameter Sets: ObjectAndDirectoryParameterSet, ObjectAndPathParameterSet
+Aliases: CloudEndpoint
+
+Required: True
+Position: 0
+Default value: None
+Accept pipeline input: True (ByValue)
+Accept wildcard characters: False
+```
+
+### -Name
+Name of the CloudEndpoint.
+
+```yaml
+Type: String
+Parameter Sets: StringAndDirectoryParameterSet, StringAndPathParameterSet
+Aliases: CloudEndpointName
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -PassThru
+{{Fill PassThru Description}}
```

Comment:
Need a description of PassThru - generally this is
> In normal execution, this cmdlet returns no value on success.  If you provide the PassThru parameter, then the cmdlet will write a value to the pipeline after successful execution

Reply:
Updated all placeholders with that message.